packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -133,6 +133,14 @@ class AnalyzeContinuously extends AnalyzeBase {
         errors.removeWhere((AnalysisError error) => error.code == 'public_member_api_docs');
         issueCount -= undocumentedMembers;
       }
+      // remove enabled experiment errors
+      final int enabledExperiments = errors.where((AnalysisError error) {
+        return error.code == 'experiment_not_enabled';
+      }).length;
+      if (argResults['permit-non-nullable'] as bool) {
+        errors.removeWhere((AnalysisError error) => error.code == 'experiment_not_enabled');
```

Comment:
Hopefully we don't need to support more than one experiment at a time ;-) Maybe the flag could have a more generic name?

Reply:
See above for more explanation here.